src/App.tsx

```diff
@@ -1,31 +1,63 @@
 import React from 'react';
 import './App.scss';
+import { getTodos } from './components/api';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
 interface State {
   selectedUserId: number;
+  todos: [];
 }
 
 class App extends React.Component<{}, State> {
   state: State = {
     selectedUserId: 0,
+    todos: [],
   };
 
+  componentDidMount() {
+    getTodos('todos')
+      .then(info => info.json())
+      .then((data: []) => {
+        this.setState({
+          todos: data,
+        });
+      });
+  }
+
+  setId = (id: number) => {
```

Comment:
```suggestion
  setUser = (id: number) => {
```
назва функції повинна описувати, що вона робить